nav2_system_tests/src/updown/test_updown_launch.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from ament_index_python.packages import get_package_prefix
+from ament_index_python.packages import get_package_share_directory
+from launch.conditions import IfCondition
+from launch.conditions import UnlessCondition
+
+import launch.actions
+
+
+def generate_launch_description():
+
+    # Configuration parameters for the launch
+
+    world = launch.substitutions.LaunchConfiguration('world')
+    params_file = launch.substitutions.LaunchConfiguration('params',
+            default=[launch.substitutions.ThisLaunchFileDir(), '/nav2_params.yaml'])
+
+    declare_world_cmd = launch.actions.DeclareLaunchArgument(
+            'world',
+            default_value=os.path.join(get_package_share_directory('turtlebot3_gazebo'), 'worlds/turtlebot3_worlds/burger.model'),
+            description='Full path to world file to load')
+
+    declare_params_file_cmd = launch.actions.DeclareLaunchArgument(
+            'params_file',
+            description='Full path to the ROS2 parameters file to use for all launched nodes')
+
+    launch_dir = os.path.join(get_package_share_directory('nav2_bringup'), 'launch')
+    gz = launch.substitutions.LaunchConfiguration('gz', default=['gzserver'])
+
+    # Specify the actions
+
+    start_gazebo_cmd = launch.actions.ExecuteProcess(
+        cmd=['gzserver', '-s', 'libgazebo_ros_init.so', world, ['__params:=', params_file]],
+        cwd=[launch_dir], output='screen')
+
+    start_robot_state_publisher_cmd = launch.actions.ExecuteProcess(
+        cmd=[
+            os.path.join(
+                get_package_prefix('robot_state_publisher'),
```

Comment:
In the system test, we got rid of the robot_state_publisher dependency as well, but since it is part of base ROS 2, it's probably OK to keep this one. If we keep it, we should have a local copy of the URDF it uses. Also, we need to add the dependency to package.xml

Reply:
Kept the dependency but introduced a local copy of the URDF file, as suggested.